### PR TITLE
Updated tweets to be in a list-group instead of a table

### PR DESCRIPTION
**Pull request by mtpultz**
Tweets were converted from being displayed in a table to a bootstrap list-group using a mobile first approach

### CODE REVIEW

File: src/app/tweets/tweets.component.html [hunk 1]
```diff
@@ -45,35 +50,50 @@ <h4 class="header-title m-t-0">All Tweets</h4>
...
-      </table>
+
+      <div class="list-group">
+        <a href="#"
+           *ngFor="let tweet of tweets$ | async | slice:0:100"
+           class="list-group-item list-group-item-action flex-column align-items-start">
+          <div class="d-flex w-100 justify-content-between">
+            <h5 class="mb-2">
+
+              <img src="assets/images/logo_postive.png"
```
Comment (jawache) [inline src/app/tweets/tweets.component.html]: Having a look at the end result when trying to make the face smaller you really can't tell if it's  red or green :(

So after second thoughts can we could turn this into some emojis instead? Maybe 👎  for negative 👍 for positive and 🤷  for unsure.

File: src/app/tweets/tweets.component.html [hunk 2]
```diff
@@ -45,35 +50,50 @@ <h4 class="header-title m-t-0">All Tweets</h4>
...
+                   width="80"
+                   height="80"
+                   alt="logo"
+                   *ngIf="tweet.sentiment < 0.33">
+
+            </h5>
+            <small>{{ timeAgo(tweet.ts) }}</small>
+          </div>
+
+          <p class="mb-2">{{tweet.text}}</p>
```
Comment (jawache) [inline src/app/tweets/tweets.component.html]: Let's try converting this to an anchor, so when you click the tweet it takes you to the actual tweet (set target="_blank" so it opens in a new page)

To get the tweet itself it's "twitter.com/statuses" + id_str

File: src/app/tweets/tweets.component.ts [hunk 3]
```diff
@@ -14,26 +14,38 @@ export class TweetsComponent implements OnInit, AfterViewInit {
...
+  ) {
     this.tweets$ = tweetService.getTweets();
   }
 
+  /**
+   * Outputs date in time ago format.
+   * @param {string} time
+   * @return {string}
+   */
+  // TODO: depending on reuse turn into a pipe or pull in package containing a pipe
```
Comment (jawache) [inline src/app/tweets/tweets.component.ts]: Agreed!

**Review by jawache — CHANGES_REQUESTED** [review]
You did exactly as the ticket described :) I just have a few more requests after seeing how it looked!

File: src/app/tweets/tweets.component.html (hunk 1 above)
Comment (mtpultz) [reply src/app/tweets/tweets.component.html]: I've made this change.  Other combinations that might work are 🙂☹😕, or maybe just switching 🤷to 😕.  I didn't immediately recognize this as uncertainty, but that doesn't mean anyone else would have that problem.

**Comment by mtpultz** [comment]
@jawache changes are ready for review

**Comment by jawache** [comment]
Awesome, thanks Looks Good To Me 👍

**Pull request merged**